config/mkpj.sh

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env bash
+# Copyright 2020 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Usage: mkpj.sh --job=foo ...
+#
+# Arguments to this script will be passed to a dockerized mkpj
```

Comment:
nit: Mention the existence of the bazel version of the script.

Also consider adding this script to the docs: https://github.com/kubernetes/test-infra/tree/master/config/jobs#kubernetes-prow-job-configs

Reply:
Is a containerized version of mkpj preferable to `bazel run //prow/cmd/mkpj`? If one method is preferable, why?

Reply:
It is, because the version of `mkpj` you use ideally should match the version you have deployed, or you risk nuking your deployment by pushing a ProwJob CR that has fields that the current controller levels don't know about.

Reply:
FWIW the bazel target I'm talking about does point to the k8s Prow instance's config: https://github.com/kubernetes/test-infra/blob/a0a273a08a4c35e150c09fd8a7f558d4cc12c793/config/BUILD.bazel#L7

I agree that the containerized version is preferable though.